manual/manual/cmds/intf-c.etex

```diff
@@ -1553,31 +1553,37 @@ to the extension of the resulting file (given with "-o").
 The native-code compiler "ocamlopt" also supports the "-output-obj"
 option, causing it to output a C object file or a shared library
 containing the native code for all OCaml modules on the command-line,
-as well as the OCaml startup code. Initialization is performed by
-calling "caml_startup" as in the case of the bytecode compiler.
+as well as the OCaml startup code and the OCaml runtime system.
+Initialization is performed by calling "caml_startup" as in the case of
+the bytecode compiler.
 
-For the final linking phase, in addition to the object file produced
-by "-output-obj", you will have to provide the OCaml runtime
-library ("libcamlrun.a" for bytecode, "libasmrun.a" for native-code),
-as well as all C libraries that are required by the OCaml libraries
+For the final linking phase, you will have to provide all C libraries
+that are required by the OCaml libraries
 used.  For instance, assume the OCaml part of your program uses the
 Unix library.  With "ocamlc", you should do:
 \begin{alltt}
         ocamlc -output-obj -o camlcode.o unix.cma {\it{other}} .cmo {\it{and}} .cma {\it{files}}
         cc -o myprog {\it{C objects and libraries}} \char92
-           camlcode.o -L`ocamlc -where` -lunix -lcamlrun
+           -L`ocamlc -where` -lunix
 \end{alltt}
 With "ocamlopt", you should do:
 \begin{alltt}
         ocamlopt -output-obj -o camlcode.o unix.cmxa {\it{other}} .cmx {\it{and}} .cmxa {\it{files}}
         cc -o myprog {\it{C objects and libraries}} \char92
-           camlcode.o -L`ocamlc -where` -lunix -lasmrun
+           -L`ocamlc -where` -lunix
 \end{alltt}
 
-% -- This seems completely wrong -- Damien
-% The shared libraries produced by "ocamlc -output-obj" or by "ocamlopt
-% -output-obj" already contains the OCaml runtime library as
-% well as all the needed C libraries.
```

Comment:
I don't understand this comment. We are not producing a shared library with `ocamlc -output-obj` we are producing a C object.